tests/get-criteria-test.js

```diff
@@ -55,12 +55,48 @@ describe('Criteria methods', function() {
         customTargetingKeyId: '597016'
       };
 
+      return expect(dfp.getCriteriaValueId(conditions))
+        .to.eventually.deep.equal('126109156816');
+    });
+
+  });
+
+  describe('getCriteriaValue', function() {
+
+    it('should return value id the same as getCriteriaValueId', function() {
+      var conditions = {
```

Comment:
It's actually different variables and they change the behavior of the test, so I'm going to keep them inside the tests